core/modules/parsers/wikiparser/rules/styleinline.js

```diff
@@ -41,14 +41,14 @@ exports.parse = function() {
 	var node = {
 		type: "element",
 		tag: "span",
-		attributes: {
-			"class": {type: "string", value: "tc-inline-style"}
-		},
 		children: tree
 	};
 	if(classString) {
```

Comment:
We should only apply "tc-inline-style" if there are no classes *or styles* assigned to the run.